### PR TITLE
Exclude Cythonized .c files from distributions

### LINKED ISSUE #284: sdist creation may include cythonized files
I had to run `python setup.py clean` to remove all cythonized `.c` files in order to not get them in the sdist. This should be fixed somehow.

### PR DESCRIPTION
**Pull request by caspervdw**
Fixes #284 

The cythonized .c files should not be included in a source distribution.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -5,4 +5,5 @@ include pygeos/_version.py
 include pygeos/*.dll
```
Comment (jorisvandenbossche) [inline MANIFEST.in:5]: Do you know what's this for?

Comment (caspervdw) [reply MANIFEST.in:5]: No, I am a bit worried about this too. It should be excluded (auditwheel injects them)

Comment (caspervdw) [reply MANIFEST.in:5]: Found it. It is a necessity for the WIndows wheelbuilder (auditwheel only works for the Linux/OSX distributions).

See https://github.com/pygeos/pygeos/pull/103

Comment (jorisvandenbossche) [reply MANIFEST.in:5]: (ah, I thought the manifest was only for sdists .. apparently not ;))

**Review by jorisvandenbossche — APPROVED** [review]

**Pull request merged**